cmd/private-org-sync/main.go

```diff
@@ -457,7 +469,7 @@ func mergeRemotesAndPush(logger *logrus.Entry, git gitFunc, repoDir, srcRemote,
 			mergeErrs = append(mergeErrs, fmt.Errorf("failed to perform merge --abort: %w", err))
 		}
 
-		return utilerrors.NewAggregate(mergeErrs)
+		return nil, utilerrors.NewAggregate(mergeErrs)
```

Comment:
So at this place, we can just log the warning and return error==nil. We don't need the second return value.